manifests/alertmanager.pp

```diff
@@ -118,11 +118,11 @@
   Boolean $manage_group                   = true,
   Boolean $manage_service                 = true,
   Boolean $manage_user                    = true,
-  String $os                              = $prometheus::os,
+  String[1] $os                           = $prometheus::os,
```

Comment:
Should this actually be `Optional[String[1]] $os = undef`? That works because `prometheus` has the right value anyway. Probably true for most of these params. Probably no reason to inherit from prometheus then either.

Reply:
this would work because prometheus::daemon.pp has a default value. I like the current approach because it provides default data for the REFERENCE.md. I'm not sure what would be the best way to continue. What are your thoughts? @alexjfisher ?

Reply:
Suggest stop using inheritance while you are focusing on this module.

Reply:
I updated it for the `$os` option. I would like to update the other params in another pull request to keep the diff here small.

Reply:
I updated the $os param. I would like to do the other in another PR to keep the diff here small.